fitbenchmarking/fitbenchmarking
# Added parameters to the problem summary page

## Description of work

Fixes #1378 




## Testing Instructions



1. run `fitbenchmarking` and verify that best fitted parameters are shown in the problem summary page.


## Checklist
